sdk/examples/wallet/10_mint_nft.rs

```diff
@@ -55,14 +64,15 @@ async fn main() -> Result<()> {
     );
 
     // Build nft output manually
-    let sender_address = account.addresses().await?[0].address().clone();
+    let account_address = &account.addresses().await?[0];
+    let sender_address = account_address.address();
     let token_supply = account.client().get_token_supply().await?;
     let outputs = vec![
         // address of the owner of the NFT
```

Comment:
This comment was already outdated it seems, should be removed or moved right above `.add_unlock_condition(AddressUnlockCondition::new(sender_address))` I would say
Maybe also changed to something like `address that will control the NFT`?